### PR TITLE
Updating InSpec Platforms

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

### Description

This PR updates the list of supported platforms for Chef InSpec to reflect support as of 4.19.

### Definition of Done

🟢  means go!

### Check List

- [x] Spell Check
- [x] Local build
- [x] Examine the local build
- [x] All tests pass

### CODE REVIEW

**Comment by james-stocks** [comment]
The table needs to reflect that we do not build for aarch64 for RHEL 6.
Otherwise this looks complete and correct! :)

File: go.mod [hunk 1]
```diff
@@ -10,4 +10,4 @@ require (
 
 //replace github.com/chef/effortless/effortless-chef-io => ../effortless
 //replace github.com/chef/chef-workstation/www => ../chef-workstation/www
-//replace github.com/chef/desktop-config/docs => ../desktop-config/docs
\ No newline at end of file
+//replace github.com/chef/desktop-config/docs => ../desktop-config/docs
```
Comment (mjingle) [inline go.mod:13]: @IanMadd I don't  know why this got highlighted....

**Review by james-stocks — APPROVED** [review]
The list is complete and correct, thank you!

**Review by IanMadd — APPROVED** [review]
tenor-225000602

**Pull request merged**